lms/views/index.py

```diff
@@ -1,8 +1,12 @@
+from pyramid.httpexceptions import HTTPFound
 from pyramid.view import view_config
 
 
-@view_config(route_name="welcome", renderer="lms:templates/base.html.jinja2")
-@view_config(route_name="index", renderer="lms:templates/base.html.jinja2")
-def index(request):  # pylint: disable=unused-argument
-    """Render an empty page that contains a needed Google verification meta tag."""
-    return {}
+@view_config(route_name="welcome")
+@view_config(route_name="index")
+def index(_):
+    """Redirect curious users to some help."""
+
+    return HTTPFound(
+        location="https://web.hypothes.is/help/installing-the-hypothesis-lms-app/"
+    )
```

Comment:
Hard coding this seems like the easiest thing to do. I don't think it's going to change, and if it does it's dirt simple to change.

If we find we are changing it every week we would think about adding an environment variable.

Reply:
This link also includes the Canadian version so we don't have to worry about two different links.